Sources/Fuzzilli/Minimization/BlockReducer.swift

```diff
@@ -84,32 +84,34 @@ struct BlockReducer: Reducer {
         verifier.tryNopping(candidates, in: &code)
     }
 
-    private func reduceGenericBlockGroup(_ group: BlockGroup, in code: inout Code, with verifier: ReductionVerifier) {
-        var candidates = group.excludingContent().map({ $0.index })
-        if verifier.tryNopping(candidates, in: &code) {
-            // Success!
-            return
+    private func reduceGenericBlockGroup(_ group: BlockGroup, in code: inout Code, with verifier: ReductionVerifier, excludingContent excludeContent: Bool = true) {
```

Comment:
What's the idea behind this change (the `excludingContent` parameter)?

Reply:
I had to introduce this for blocks that beign with `BeginClassDef`, `BeginWith` and `BeginAsync/Generator*` operations. For these specific blocks, Noping out the Begin and End operations more often leads to statically invalid Programs since they provision special contexts for some operations to exist in (eg. `BeginMethodDef` under a `BeginClassDef`). So it made sense to me to skip this initial reduction step and jump right to reduction that takes into consideration the block content.